src/kvm/mod.rs

```diff
@@ -504,6 +543,51 @@ impl Hypervisor {
         })
     }
 
+    pub fn transfer(&self, fds: &[RawFd]) -> Result<Vec<RawFd>> {
+        let addr_local_mem = self.alloc_mem()?;
+        let addr_remote_mem = self.alloc_mem()?;
+        let msg_hdr_mem = self.alloc_mem()?;
+        let iov_mem = self.alloc_mem()?;
+        let iov_buf_mem = self.alloc_mem::<[u8; 1]>()?;
+        let cmsg_mem = self.alloc_mem::<[u8; 64]>()?; // should be of size CMSG_SPACE, but thats not possible at compile time
+
+        let ret;
+        let hv;
+        {
+            let tracee = try_with!(
+                self.tracee.write(),
+                "cannot obtain tracee write lock: poinsoned"
+            );
+            let vmsh_id: u64 = rand::thread_rng().gen::<u64>();
```

Comment:
Could we not just the process id here? It should be unique.

Reply:
addressed in 554ff3d. SockAddr now looks like `vmsh_fd_transfer_<pid>`.